llvm/test/MC/ELF/rtti-proxy-gotpcrel.ll

```diff
@@ -1,4 +1,6 @@
-; RUN: llc %s -mtriple=x86_64-unknown-fuchsia  -o - | FileCheck %s
+; RUN: llc %s -mtriple=x86_64-unknown-fuchsia -o - | FileCheck %s
+; RUN: llc %s -mtriple=aarch64-unknown-fuchsia -o - | FileCheck %s
```

Comment:
MC/ELF requires X86.

For aarch64 and riscv64, you need `%if aarch64-registered-target` and a similar directive for riscv.